pkg/nvlib/info/info.go

```diff
@@ -87,6 +131,56 @@ func (i *infolib) IsTegraSystem() (bool, string) {
 	return false, fmt.Sprintf("%v has no 'tegra' prefix", tegraFamilyFile)
 }
 
+// UsesNVGPUModule checks whether the nvgpu module is used.
+// We use the device name to signal this, since devices that use the nvgpu module have their device
+// names as:
+//
+//	GPU 0: Orin (nvgpu) (UUID: 54d0709b-558d-5a59-9c65-0c5fc14a21a4)
+//
+// This function returns true if ALL devices use the nvgpu module.
```

Comment:
What exactly is the `nvgpu` module?  Is this the name of the kernel module for iGPUs? Maybe mention that here somehwere if so.

Reply:
Yes, that is the kernel module for igpus. I can update the docstring.

Reply:
I have updated this in the latest. Please take another look.